creator-node/src/routes/tracks.js

```diff
@@ -454,6 +455,9 @@ module.exports = function (app) {
    * @dev - Wrapper around getCID, which retrieves track given its CID.
    **/
   app.get('/tracks/stream/:blockchainId', async (req, res) => {
+    const libs = req.app.get('audiusLibs')
+    const delegateOwnerWallet = config.get('delegateOwnerWallet')
```

Comment:
Not sure if this was the best way to track "which creator node a listen comes from"

what do you guys think?

Reply:
i believe spID is a better way cc @hareeshnagaraj ?

Reply:
yes spID is a better way - there will not be a 1:1 delegateWallet to spID in our. next eth contracts iteration

Reply:
resolved offline